keel-core-test/src/main/kotlin/com/netflix/spinnaker/keel/persistence/DeliveryConfigRepositoryTests.kt

```diff
@@ -490,8 +490,12 @@ abstract class DeliveryConfigRepositoryTests<T : DeliveryConfigRepository, R : R
         test("can retrieve the manifest for the resources") {
           val resource = deliveryConfig.resources.random()
           val persistedDeliveryConfig = repository.deliveryConfigFor(resource.id)
+            .let {
+              // disregard `createdAt` which is added when reading from the database
+              it.copy(metadata = it.metadata.filterKeys { k -> k != "createdAt" })
+            }
```

Comment:
Isn't this what you're testing? Why do you disregard it?

Reply:
I've revisited the test per your suggestion in chat.